src/frontend/src/markdown-pages/about.md

```diff
@@ -0,0 +1,69 @@
+---
+path: '/about'
+date: '2019-05-04'
+title: 'About'
+---
+
+# Telescope Overview
+
+## Introduction
+
+One of the key features of Seneca's open source involvement has been the
+emphasis on sharing what we're working on, teaching, and learning through blogging.
+We believe that one of the most rewarding parts of learning to work in the
+open source community is discovering that one can become part of the fabric
+of the web, find a voice, and build a following.
+
+We also believe that reading each other's blog posts is important. In the blog
+posts of our colleagues, we find that we are not alone in our struggles to make
+things work, our interests in various topics, and that imposter syndrome isn't
+something unique to "me."
+
+To better enable the discovery of blogs within our community, we set up an open
+source blog [Planet](<https://en.wikipedia.org/wiki/Planet_(software)>): an aggregated
+feed of blog posts from Seneca faculty and students working on open source in a
+single page. Our blog Planet currently lives at [http://zenit.senecac.on.ca/~chris.tyler/planet/](http://zenit.senecac.on.ca/~chris.tyler/planet/).
```

Comment:
This link to the old planet is dead. Let's update this sentence to read:

```
Our old blog Planet used to live at http://zenit.senecac.on.ca/~chris.tyler/planet/, and was run faithfully by Chris Tyler for more than a decade.  We've made a re-creation of what it looked like at [http://telescope.cdot.systems/planet](http://telescope.cdot.systems/planet) if you want to see it.
```

This links to our production page which isn't available yet.  Is there some way in Gatsby for you to swap in this link via script when building this page, so we can have staging vs. production?

Reply:
I'm not sure how to swap links but I updated the text here :)